auth-web/src/util/constants.ts

```diff
@@ -41,7 +41,9 @@ export enum Pages {
 }
 
 export enum Account {
-    ANONYMOUS = 'ANONYMOUS'
+    ANONYMOUS = 'ANONYMOUS',
```

Comment:
Use this enum where you can.  Extend it with 'IMPLICIT' / 'EXPLICIT' until we remove those.